game/view/sector.lua

```diff
@@ -211,6 +214,37 @@ function SectorView:getBodySprite(body)
   return body_sprite
 end
 
+
+function SectorView:setBodySprite(body, draw)
+  self.body_sprites[body:getId()] = draw
```

Comment:
WTF is draw? Please give it a better name

Reply:
Don't know when I created this or for what reason since no one calls this. Removing it.